web/src/js/ducks/ui/flow.js

```diff
@@ -148,7 +148,6 @@ export function setContent(content){
     return { type: SET_CONTENT, content }
 }
 
-export function stopEdit(data, modifiedFlow) {
-    let diff = getDiff(data, modifiedFlow)
-    return {type: flowsActions.UPDATE, data, diff }
+export function stopEdit(flow, modifiedFlow) {
+    return flowsActions.update(flow, getDiff(flow, modifiedFlow))
```

Comment:
This is explained here: https://github.com/mhils/mitmproxy/blob/c4804b5fe10e9ca13380cd5b9815e71ffc0d64d3/web/src/js/ducks/ui/flow.js#L60